src/babel/__tests__/examples/stylesheet.js

```diff
@@ -35,4 +39,10 @@ const styles = zacs._experimentalStyleSheet({
       color: #abbaba;
     }`,
   },
+  css: `
```

Comment:
I'd suggest adding a mock CSS literal factory for cases like this one, eg.:

```js
import zacs from '@nozbe/zacs'

const styles = zacs.createStyleSheet({
  root: {
    css: zacs.css`
       background: red;
    `
  }
})
``` 

This way you get syntax highlighting in VSC (with some plugins ofc) and formatting the CSS inside (eg. with Prettier) for free

Reply:
excellent pro tip, I'll do this :)